### DISCUSSION TITLE
Terrain and Layer rendering problem

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1947] [Support Q&A] [answered]
Is there anyone who knows why the terrain is cut when the pitch attitude as depicted in the picture and close the the terrain?

![image](https://user-images.githubusercontent.com/5169314/151676990-5851273c-ee13-4be8-864f-efd1f2a69c57.png)

I solved to problem with : viewer->getCamera()->setNearFarRatio(0.000001);

But this time, my models show weird look as in below picture. I think this is because I set the setNearFarRatio value too low. But without the above value I see the same ground clipping. What should I do?

![image](https://user-images.githubusercontent.com/5169314/151743763-a9cd3052-42e6-49bd-a2dc-54de87bf7298.png)

**Comment by gwaldron** [comment, marked as answer]
Which version are you using?
Looks like you need to enable the logarithmic depth buffer on your camera:
```c++
LogarithmicDepthBuffer::install(camera);
```

**Reply by nebsar** [reply]
I am using osgEarth 3.2. Thanks Glenn.
